phoenix-pig-base/phoenix4-pig-shaded/pom.xml

```diff
@@ -68,5 +68,16 @@
       <artifactId>phoenix-hbase-compat-${hbase.compat.version}</artifactId>
       <scope>runtime</scope>
     </dependency>
+    <!-- We want to take the implementation from Pig -->
```

Comment:
Frankly I'm not sure if the pig shaded JAR even works, much less if this would make it better or worse.